### DISCUSSION TITLE
G-Helper Headless

### DISCUSSION THREAD

**Discussion by yesrab** [seerge/g-helper#1216] [Ideas]
Are there any plans for a headless version of this app so that people could come up with their own UI elements and frontend
or an ability to modify the existing front with theming implementation.

**Comment by seerge** [comment]
I have no idea what "headless" means, but you can always fork app and adjust UI for yourself if you want to.

**Reply by ke1ne** [reply]
It sounds like some "console" exe to run with argumens, so third-party UI can be built to send them.

**Reply by seerge** [reply]
This is not how app works in a first place, it has too many automatic actions that run when certain events happen.

**Reply by ke1ne** [reply]
> This is not how app works in a first place, it has too many automatic actions that run when certain events happen.

Seeing no reason too. To make some custom app, a full source is already available.

**Reply by yesrab** [reply]
i see well thanks for responding 
i will take a look at the source and see what i can do but its gonna take some time since im not so sharp in C#

**Reply by seerge** [reply]
I'm not c# developer neither, I'm not even windows developer ... :)

**Reply by yesrab** [reply]
by Headless i mean the application runs without a GUI and people could build their own GUI 
ofc there would me some sort of communication b/w the headless componet and the GUI for example a API of sorts
